BDPointiOSXamarinDemo/ViewController.cs

```diff
@@ -83,6 +86,14 @@ private void InitializeSDK()
 
         }
 
+        private void updateVersionLabels()
+        {
+            var appVersion = NSBundle.MainBundle.InfoDictionary[new NSString("CFBundleShortVersionString")].ToString();
+            var buildNumber = NSBundle.MainBundle.InfoDictionary[new NSString("CFBundleVersion")].ToString();
+            appVersionLabel.Text = "App: " + appVersion + "(" + buildNumber + ")";
+            sdkVersionLabel.Text = "SDK: " + locationManager.SdkVersion();
+        }
```

Comment:
could we have a ReleaseNotes file and add this new change to that?

Reply:
Sure